### PR TITLE
Added LittleTable to bigtable emulators

### PR DESCRIPTION
**Pull request by steveniemitz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ A curated list of tools that will help you when working with or building on-top
 - [Google Emulator](https://cloud.google.com/bigtable/docs/emulator) - Official in-memory emulator for Cloud Bigtable, included with the Google Cloud SDK.
 - [Spotify Docker Bigtable](https://github.com/spotify/docker-bigtable) - Docker container with an in memory implementation of Google Cloud Bigtable.
 - [Shopify Bigtable Emulator](https://github.com/Shopify/bigtable-emulator) - In memory Go implementation of Bigtable.
+- [Littletable](https://github.com/steveniemitz/littletable) - In-memory JVM-based emulator for Bigtable.
```
Comment (mbrukman) [inline README.md]: s/Littletable/LittleTable/ since I think that's how you capitalize it in the docs? Otherwise, LGTM. Thanks!

**Review by mbrukman — CHANGES_REQUESTED** [review]

**Comment by zrosenbauer** [comment]
@steveniemitz you want to update your PR so we can get this in?

**Comment by steveniemitz** [comment]
oh thanks for the ping, I had totally forgotten about this :)

**Review by mbrukman — APPROVED** [review]
LGTM

**Comment by mbrukman** [comment]
Thanks for the PR, @steveniemitz!

**Pull request merged**